src/scripts/main.js

```diff
@@ -5,4 +5,17 @@ const people = require('./lib/people');
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const table = document.querySelector('tbody');
+
+people.forEach(el => {
```

Comment:
What does el mean? Give more meaningful names 
```suggestion
people.forEach(person => {
```